### ISSUE TITLE
Text wird abgeschnitten, wenn seine Größe verändert wird.

### ISSUE THREAD

**Issue by xwolfde** [RRZE-Webteam/FAU-Einrichtungen#1285]
Betrifft neuen Slider: Titel. Dieser hat wohl ein overflow hidden.
Besucher müssen in der Lage sein, Text auf 200 % zu skalieren, ohne Informationen zu verlieren.

<img width="1537" alt="grafik" src="https://user-images.githubusercontent.com/1070669/236770282-9a23f585-0316-49bc-b90b-e9ef3d28c1ee.png">

**Comment by xwolfde** [comment]
Ausserdem berechtiger Feedback aus der Nutzerseite: Es muss bekannt und klar sein, wieviele Zeichen man da setzen kann. Nur dann kann man auch darauf redaktionell eingehen.

**Comment by FitoreGashi** [comment]
Es werden immer 62 Zeichen angezeigt und die anderen werden auf einem größeren Bildschirm angezeigt. Die Idee des Designs bestand darin, den Text mit ... auszuschneiden, damit der Benutzer weiß, ob er den vollständigen Titel anklicken möchte
 @danielboese

**Comment by xwolfde** [comment]
Naja, wir können generell für alle ab einer gewissen Anzahl an Zeichen schneiden. Nur eben nicht optisch: Im Sinne der Barrierefreiheit ist wichtig, dass jeden denselben Content bekommt und es nicht optisch geschnitten wird, nur weil jemand z.B. 200% Schriftgröße hat.

Vgl: https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-scale.html

**Comment by FitoreGashi** [comment]
![Screenshot 2023-05-31 at 12 46 49](https://github.com/RRZE-Webteam/FAU-Einrichtungen/assets/101202370/ed5adf95-252b-446a-b6f1-d6f2ff70fe5a)
![Screenshot 2023-05-31 at 12 46 31](https://github.com/RRZE-Webteam/FAU-Einrichtungen/assets/101202370/55d08fe1-fc6b-4edb-8fe1-83fd987fcea5)
Ja, hier ist ein Beispiel, bei dem 62 Zeichen ebenfalls auf 200 % angezeigt werden, aber auf zwei Zeilen, sodass 62 die Zahl ist, die in beiden Szenarien in der Desktop-Ansicht vollständig angezeigt wird

**Comment by xwolfde** [comment]
Passt, aber Achtung: Gebe dem Bild man eine längere Copyright-Info. Dann überlagert der blaue Bereich die Copyright-Info.

**Comment by AndiWebred** [comment]
Für die Startseite bzw. den Slider erhalten wir weiterhin eine Fehlermeldung bei Siteimprove, dass der TExt abgeschnitten wird.
Siehe: https://my2.siteimprove.com/Inspector/705695/A11Y/Page?pageId=74034471241&impmd=CDF7C5F66B15D04FC961306C8B003AD3&conf=a+aa+aaa+si#/sia-r83/failed/dXBPV00rRUY=;MDY4OTlmZTI=
